site/content/en/docs/handbook/troubleshooting.md

```diff
@@ -19,15 +19,15 @@ Example:
 
 ## Post-mortem minikube debug logs
 
-minikube stores post-mortem INFO logs in the temporary directory of your system. On macOS or Linux, it's easy to get a list of recent INFO logs:
+minikube stores post-mortem logs in the temporary directory of your system. One logfile is created per subcommand and any subsequent invocations of the subcommand with the same args will append to the same file. On macOS or Linux, it's easy to get a list of recent logs:
```

Comment:
I think the `One log file per subcommand` is slightly misleading since once the file is over a MB a new file is created. Also, need a space between `logfile`.

Reply:
Added a line to capture this behavior